_install-and-configure/install-opensearch/index.md

```diff
@@ -90,3 +90,11 @@ The [sample docker-compose.yml]({{site.url}}{{site.baseurl}}/install-and-configu
 Do not declare the same JVM options in multiple locations because it can result in unexpected behavior or a failure of the OpenSearch service to start. If you declare JVM options using an environment variable, such as `OPENSEARCH_JAVA_OPTS=-Xms3g -Xmx3g`, then you should comment out any references to that JVM option in `config/jvm.options`. Conversely, if you define JVM options in `config/jvm.options`, then you should not define those JVM options using environment variables.
 {: .note}
 
+## Important system properties
+
+The OpenSearch has a number of system properties that you could specify in `config/jvm.options` or `OPENSEARCH_JAVA_OPTS`, using `-D` command line arguments notation.
+
+- `opensearch.xcontent.string.length.max=<value>`
```

Comment:
Could we add an introductory sentence for this snippet? How about this:
````suggestion
For example, the following setting sets the `opensearch.xcontent.string.length.max` to `<value>`:

```plaintext
- opensearch.xcontent.string.length.max=<value>
```
````

Reply:
My intention here was to keep it as a lit, similarly to the previous section (not changed here) with system parameters.

Reply:
Understood. See my suggestion above.